pkg/agent/manager/cache/lru_cache.go

```diff
@@ -283,8 +298,11 @@ func (c *LRUCache) UpdateEntries(update *UpdateEntries, checkSVID func(*common.R
 			delete(c.staleEntries, id)
 		}
 	}
+	c.metrics.IncrCounter([]string{EntryRemoved}, float32(entriesRemoved))
```

Comment:
all metric definitions are centralized to the `telemetry` package https://github.com/spiffe/spire/tree/main/pkg/common/telemetry / https://github.com/spiffe/spire/blob/main/doc/telemetry.md